index.js

```diff
@@ -0,0 +1,30 @@
+'use strict'
+
+const fp = require('fastify-plugin')
+const Express = require('express')
+
+function expressPlugin (fastify, options, next) {
+  const express = Express()
+
+  // TODO: we should use decorate, but `use` is already
+  // a public API of Fastify. In Fastify v3 it will be deprecated,
+  // so we will able to use `decorate`
+  fastify.use = use
+  fastify.addHook('onRequest', runConnect)
+
+  function use (fn) {
+    express.use(fn)
```

Comment:
This is missing the path prefix feature:

```js
fastify.use('/prefix', function middleware (req, res, next) { })
```

Reply:
Thanks!